src/gsy_e/models/strategy/external_strategies/__init__.py

```diff
@@ -316,6 +320,117 @@ def _get_market_from_time_slot(self, time_slot: DateTime) -> MarketBase:
                 f"settlement markets")
         return market
 
+    def _offer_aggregator_impl(
```

Comment:
I am leaving the comment here in order to ideally not discuss it in this PR since we have discussed it already: I think that moving this method to its own offer mixin would be good in the long run, however it should not be done in the context of this ticket since we have another open ticket that deals with this.

Reply:
I have a followup question though: why was it necessary to be moved in the ExternalMixin in the context of this PR ? I assume to avoid code duplication, right ?

Reply:
Yes, this was my try to be DRY and to re-use this code in the external PV and in the Load.